mw-server/schemas/problems.schema.go

```diff
@@ -23,3 +25,14 @@ type ProblemPopulatedResponse struct {
 	DayReportUuid string           `json:"dayReportUuid" validate:"required"`
 	Tags          []JobTagResponse `json:"tags" validate:"required"`
 }
+
+type ProblemPopulatedDTO struct {
```

Comment:
very dangerous - after changes in sql the will be 100% error in runrime


probably we should fix root cause - write format for date types